client/js/game.js

```diff
@@ -9356,7 +9357,8 @@ function Game() {
                     text(ctx,"Tournament hadn't finished",xpos,H*0.5,"26px"+FONT,"black","center","middle");
                 } else {
                     var yprice=TPRICE[tdata[showDay].tid%7];
-                    if (mdata.city.tour[tpos].tid<=17349&&yprice==0) yprice=50;
+                    //if (mdata.city.tour[tpos].tid<=17349&&yprice==0) yprice=50;
+					if (yprice==0) yprice=50;
                     var yourReward=tprize(mdata.city.tour[tpos].top,mdata.city.tour[tpos].amount,yprice);
                     //var pranas=TPG[yprice];
                     var pranas=TPG[yprice];
```

Comment:
WIth the changed yprice this should yield the wrong Prana again... Maybe change TPG or only fiy yprice==0 inside of tprice